clients.asciidoc

```diff
@@ -340,35 +340,64 @@ Geth and Parity can only operate fast synchronization when starting with an empt
 
 == Light Clients
 
-=== MyEtherWallet
+Lightweight Ethereum clients are clients that offer a subset of the functionality of a full client. They do not store the full Ethereum blockchain, so they are faster to setup and require far less data storage.
 
-=== MyCrypto
+A lightweight client offers one or more of the following functions:
 
-=== BlockCypher
+* A wallet that can manage private keys and Ethereum addresses.
+* The ability to create, sign and broadcast transactions.
+* The ability to interact with smart contracts, using the data payload.
+* A web3 library, injected into the web browser as a JavaScript object.
```

Comment:
IMHO, here we have two different types of lighweight clients, 

- The **injectors** of "a **web3 object** and convenience **Web3.js library** into the **javascript context**", e.g. **"Metamask"**.
- And those that use the previously injected object and/or Web3.js library, like **MEW** and **MyCrypto**.

As lightweight client functions I'd say;

- **web3 object** injection and convenience **Web3.js library**  into the **javascript context**.
- Use the previously **injected web3 object** and/or Web3.js library (the best practice is to explicitly bundle the version of web3.js you used during development)